db/migrations/1723836233391-Data.js

```diff
@@ -7,8 +7,8 @@ module.exports = class Data1723836233391 {
         await db.query(`DROP INDEX "public"."IDX_9415c52ccc157e1a37c9f3f623"`)
         await db.query(`ALTER TABLE "send_packet_fee_deposited" RENAME COLUMN "send_gas_limit" TO "recv_gas_limit"`);
         await db.query(`ALTER TABLE "send_packet_fee_deposited" RENAME COLUMN "send_gas_price" TO "recv_gas_price"`);
-        await db.query(`ALTER TABLE "send_packet" ADD "total_recv_fees_deposited" numeric NOT NULL`)
-        await db.query(`ALTER TABLE "send_packet" ADD "total_ack_fees_deposited" numeric NOT NULL`)
+        await db.query(`ALTER TABLE "send_packet" ADD "total_recv_fees_deposited" numeric`)
+        await db.query(`ALTER TABLE "send_packet" ADD "total_ack_fees_deposited" numeric`)
```

Comment:
Why do you need to make it nullable? Is it possible for those fields to have null values?

Reply:
My thinking was
1. Solve the error being thrown on deploy
2. Could set a default value of 0, but that implies we know the no fees have been deposited. In the case that a fee is missed for some reason it seems more fitting that the associated sendPacket has these values set to null.